clarifai/datasets/upload/loaders/README.md

```diff
@@ -4,43 +4,46 @@ A collection of data preprocessing modules for popular public datasets to allow
 
 ## Usage
 
-If a dataset module exists in the zoo, uploading the specific dataset can be easily done by simply creating a python script (or via commandline) and specifying the dataset module name in the `dataset_loader` parameter of the  `Dataset` class, `upload_dataset` method .i.e.
+If a dataset module exists in the zoo, uploading the specific dataset can be easily done by simply creating a python script (or via commandline) and specifying the dataloader object in the `dataloader` parameter of the  `Dataset` class, `upload_dataset` method .i.e.
 
 ```python
 from clarifai.client.app import App
+from clarifai.datasets.upload.loaders.coco_segmentation import COCOSegmentationDataLoader
 
 app = App(app_id="", user_id="")
 # Create a dataset in Clarifai App
 dataset = app.create_dataset(dataset_id="")
+# instantiate dataloader object
+coco_seg_dataloader = COCOSegmentationDataLoader(images_dir="", label_filepath="")
 # execute data upload to Clarifai app dataset
-dataset.upload_dataset(task='visual_segmentation', split="train", dataset_loader='coco_segmentation')
+dataset.upload_dataset(dataloader=coco_seg_dataloader)
 ```
 
 ## Dataset Loaders
 
- | dataset name | task | module name (.py) | splits |
- | --- | --- | --- | --- |
- | [COCO 2017](https://cocodataset.org/#download) | Detection | `coco_detection` | `train`, `val` |
- |        | Segmentation | `coco_segmentation` | `train`, `val` |
- |       | Captions | `coco_captions` | `train`, `val` |
- |[xVIEW](http://xviewdataset.org/)  | Detection | `xview_detection` | `train`
- | [ImageNet](https://www.image-net.org/)  | Classification | `imagenet_classification` | `train`
+ | dataset name | task | module name (.py)
+ | --- | --- | ---
+ | [COCO 2017](https://cocodataset.org/#download) | Detection | `coco_detection` |
+ |        | Segmentation | `coco_segmentation` |
+ |       | Captions | `coco_captions` |
```

Comment:
If these are not yet supported, maybe we can open up issues in the repo with "help wanted" / "good first issue" tag if the team does not plan on taking this up in the near future.

Reply:
Good idea. But for now these are supported

Reply:
I see. I got confused as the cells for seg/captions were empty under "dataset name". Is it possible to merge those cells together for those 3 rows for that column?

Reply:
I don't think we can merge cells using plain markdown. We'll have to use custom html tags

Reply:
That's too bad. Let's leave it then.